api/apps/articles/model/schema.coffee

```diff
@@ -56,6 +58,11 @@ denormalizedArtwork = (->
         name: @string().allow('', null)
         slug: @string().allow('', null)
     ).allow(null).default([])
+    artist: @object().keys
+      name: @string().allow('', null)
+      slug: @string().allow('', null)
```

Comment:
Adding back in singular artist for now because it looks like ImageSet is broken on staging